src/cli.js

```diff
@@ -14,54 +14,65 @@ program
     .option('--carebear', 'Do not care about style guide errors. Shame on you.')
     .option('--optimize', 'Remove debug checks for types (types are experimental)')
     .option('--eval [input]', '')
+    .option('-c, --config [input]', 'Configs you want to run, seperated by comma.')
     .option('--stdio', '')
     .parse(process.argv);
 
-var options = {
-    node: !!program.node,
-    removeComments: !!program['remove-comments'],
-    strictStyleMode: !program['carebear'],
-    optimize: !!program['optimize'],
-    bare: !!program.bare,
-};
-
-if (program.stdio) {
-    process.stdin.setEncoding('utf8');
+runTasks(program);
 
-    process.stdin.on('readable', function() {
-        parseStringOrPrintError(process.stdin.read() || '');
-    });
+function runTasks(program) {
+    var baileyrcPath = process.cwd() + '/.baileyrc';
+    var options = {
+        node: false,
+        bare: false,
+        removeComments: false,
+        strictStyleMode: true,
+        optimize: false,
+        config: 'default'
+    };
 
-    process.stdin.resume();
-    return;
-}
-
-if (program.eval) {
-    return parseStringOrPrintError(program.eval);
+    if (fs.existsSync(baileyrcPath) && program.args.length == 0) {
```

Comment:
Should probably use a triple equals here? Or just `!program.args.length` etc.

Also, if you swap the order of the checks it won't have to do the `fs`-call if an argument is given :speedboat:.